pyfar/_concepts/filter_types.py

```diff
@@ -75,7 +80,11 @@
    :width: 50%
    :alt: Cross-over contained in pyfar
 
-.. |eqs| image:: resources/filter_types_parametric-eq.png
-   :width: 100%
-   :alt: Parametric equalizer contained in pyfar
```

Comment:
The corresponding figure should be removed then, right?

Reply:
Hm, it was already removed in my version. Maybe I did not push it or you looked at an outdated version.

Reply:
Hm, now I'm confused. Here https://github.com/pyfar/pyfar/tree/feature/notch_filter/docs/concepts/resources the files are still shown?

Reply:
Oh sorry, I must have looked at something else. I think I kept the old version at first, in case we don't agree on the new version. It's deleted now.